src/styles/theme/oc-text.scss

```diff
@@ -13,7 +13,7 @@ a {
   color: var(--oc-color-swatch-primary-default);
 
   &:hover {
-    color: var(--oc-color-swatch-primary-hover);
+    color: var(--oc-color-swatch-brand-hover);
```

Comment:
You modify the `swatch-primary-hover` (from my understanding) in the color.yaml file but then change the used variable. Does that mean we don't need the adjustment for the token?

Reply:
good spot @pascalwengerter 
@stevefrenzel please stay within the same color group if possible. text color from `primary` group and hover color from `brand` group would result in a theming mess, semantically.